README.md

```diff
@@ -35,17 +52,66 @@ package in the image.
 
 `openhpc_cluster_name`: name of the cluster
 
-`openhpc_packages`: additional OpenHPC packages to install
+#### Accounting
 
-`openhpc_enable`:
-* `control`: whether to enable control host
-* `batch`: whether to enable compute nodes
-* `runtime`: whether to enable OpenHPC runtime
-* `drain`: whether to drain compute nodes
-* `resume`: whether to resume compute nodes
+By default, the accounting plugin will use the `accounting_storage/filetxt` storage type. However,
+this only supports a subset of `sacct` commands.
+
+To deploy and configure `slurmdbd`:
+
+* Configure a mariadb or mysql server as documented in the slurm accounting (documentation)[https://slurm.schedmd.com/accounting.html]
```

Comment:
From this and the following point, it's not clear to me if they have to be the same node? If so I'd suggest replacing both with:
 
- Configure a mariadb or mysql server as described in the slurm accounting [documentation](https://slurm.schedmd.com/accounting.html) on one of the nodes in your inventory and set `openhpc_enable.database` to `true` for this node.

If not, just fix the doc link markup/works as above anyway!